pkg/cloud/ibm/assets/deployment.yaml

```diff
@@ -0,0 +1,132 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ibm-cloud-controller-manager
+  namespace: openshift-cloud-controller-manager
+  labels:
+    k8s-app: ibm-cloud-controller-manager
+spec:
+  replicas: 2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+      maxSurge: 3
```

Comment:
Max surge of 3 doesn't make sense if you can only have 2 replicas.  This should be at most 2 now.

Reply:
updated